tools/sgen/input/FluentUIStyle.yml

```diff
@@ -0,0 +1,67 @@
+  Foreground: [ neutral1: "FluentUIColor(light: $Colors.Neutral.grey14, lightHighContrast: $Colors.Neutral.black, dark: $Colors.Neutral.white)", neutral2: "FluentUIColor(light: $Colors.Neutral.grey26, lightHighContrast: $Colors.Neutral.black, dark: $Colors.Neutral.grey84, darkHighContrast: $Colors.Neutral.white)", neutral3: "FluentUIColor(light: $Colors.Neutral.grey38, lightHighContrast: $Colors.Neutral.grey14, dark: $Colors.Neutral.grey68, darkHighContrast: $Colors.Neutral.grey84)", neutral4: "FluentUIColor(light: $Colors.Neutral.grey50, lightHighContrast: $Colors.Neutral.grey26, dark: $Colors.Neutral.grey52, darkHighContrast: $Colors.Neutral.grey84)", neutralDisabled: "FluentUIColor(light: $Colors.Neutral.grey74, lightHighContrast: $Colors.Neutral.grey38, dark: $Colors.Neutral.grey36, darkHighContrast: $Colors.Neutral.grey62)", neutralInverted: "FluentUIColor(light: $Colors.Neutral.white, dark: $Colors.Neutral.black)", brandRest: "FluentUIColor(light: $Colors.Brand.primary, lightHighContrast: $Colors.Brand.shade20, dark: $Colors.Brand.primary, darkHighContrast: $Colors.Brand.tint20)", brandHover: "FluentUIColor(light: $Colors.Brand.shade10, dark: $Colors.Brand.shade10)", brandPressed: "FluentUIColor(light: $Colors.Brand.shade30, dark: $Colors.Brand.shade30)", brandSelected: "FluentUIColor(light: $Colors.Brand.shade20, dark: $Colors.Brand.shade20)", brandDisabled: "FluentUIColor(light: $Colors.Neutral.grey74, dark: $Colors.Neutral.grey36)" ]
+  Stroke: [ neutral1: "FluentUIColor(light: $Colors.Neutral.grey94, lightHighContrast: $Colors.Neutral.grey38, dark: $Colors.Neutral.grey24, darkHighContrast: $Colors.Neutral.grey68, darkElevated: $Colors.Neutral.grey32)", neutral2: "FluentUIColor(light: $Colors.Neutral.grey88, lightHighContrast: $Colors.Neutral.grey38, dark: $Colors.Neutral.grey32, darkHighContrast: $Colors.Neutral.grey68, darkElevated: $Colors.Neutral.grey36)", neutral3: "FluentUIColor(light: $Colors.Neutral.grey82, lightHighContrast: $Colors.Neutral.grey38, dark: $Colors.Neutral.grey40, darkHighContrast: $Colors.Neutral.grey68, darkElevated: $Colors.Neutral.grey52)", neutralDisabled: "FluentUIColor(light: $Colors.Neutral.grey88, dark: $Colors.Neutral.grey32)", neutralAccessible: "FluentUIColor(light: $Colors.Neutral.grey38, lightHighContrast: $Colors.Neutral.grey14, dark: $Colors.Neutral.grey68, darkHighContrast: $Colors.Neutral.grey84, darkElevated: $Colors.Neutral.grey68)", brandRest: "FluentUIColor(light: $Colors.Brand.primary, dark: $Colors.Brand.primary)", brandHover: "FluentUIColor(light: $Colors.Brand.shade10, dark: $Colors.Brand.shade10)", brandPressed: "FluentUIColor(light: $Colors.Brand.shade30, dark: $Colors.Brand.shade30)", brandSelected: "FluentUIColor(light: $Colors.Brand.shade20, dark: $Colors.Brand.shade20)", brandDisabled: "FluentUIColor(light: $Colors.Neutral.grey88, dark: $Colors.Neutral.grey84)" ]
```

Comment:
instead this one line can we make it more readable by having separate lines?

Reply:
+1 there is yaml formatter, so instead of 

```yml
Colors:
  Brand: [ primary: "FluentUIColor(light: #0078D4, dark: #0086F0)", shade10: "FluentUIColor(light: #106EBE, dark: #1890F1)", shade20: "FluentUIColor(light: #005A9E, dark: #3AA0F3)", shade30: "FluentUIColor(light: #004578, dark: #6CB8F6)", tint10: "FluentUIColor(light: #2B88D8, dark: #0078D4)", tint20: "FluentUIColor(light: #C7E0F4, dark: #004C87)", tint30: "FluentUIColor(light: #DEECF9, dark: #043862)", tint40: "FluentUIColor(light: #EFF6FC, dark: #092C47)" ]
```

we can view it as 
```yml
Colors:
  Brand:
    - primary: 'FluentUIColor(light: #0078D4, dark: #0086F0)'
    - shade10: 'FluentUIColor(light: #106EBE, dark: #1890F1)'
    - shade20: 'FluentUIColor(light: #005A9E, dark: #3AA0F3)'
    - shade30: 'FluentUIColor(light: #004578, dark: #6CB8F6)'
    - tint10: 'FluentUIColor(light: #2B88D8, dark: #0078D4)'
    - tint20: 'FluentUIColor(light: #C7E0F4, dark: #004C87)'
    - tint30: 'FluentUIColor(light: #DEECF9, dark: #043862)'
    - tint40: 'FluentUIColor(light: #EFF6FC, dark: #092C47)'
```

[Here](https://jsonformatter.org/yaml-formatter) is one formatter that I have used for indenting.